presto-orc/src/main/java/com/facebook/presto/orc/metadata/ExceptionWrappingMetadataReader.java

```diff
@@ -114,7 +112,6 @@ public StripeFooter readStripeFooter(List<OrcType> types, InputStream inputStrea
 
     private OrcCorruptionException propagate(Throwable throwable, String message)
     {
-        propagateIfPossible(throwable, PrestoException.class);
```

Comment:
Why this line get removed as well? -- I have seen `IOException` (e.g. connection reset when reading from storage system) are now treated as `HIVE_BAD_DATA`  due to Presto thought it's `OrcCorruptionException`? 

cc @NikhilCollooru , @highker

Reply:
I see. Because we want to drop `PrestoException` class from `presto-orc`. But in that case shouldn't we figure out some way to propagate it (without referring to `PrestoException`)?